e2-studio-star-rx72n-firmware/src/tasks/motor_control_task.c

```
@@ -2858,13 +2837,11 @@ static void internal_update_motor_state(void)
  */
 static float internal_get_target_velocity(const motor_command_t* cmd, uint8_t motor_idx)
 {
-  float target;
-
   if (cmd == nullptr || motor_idx >= k_motor_count) {
     return 0.0f;
   }
 
-  target = cmd->target_velocity_mps[motor_idx];
+  float target = cmd->target_velocity_mps[motor_idx];
```

Clang (14.0.6)

[warning] variable 'target' is not initialized

(cppcoreguidelines-init-variables)